### ISSUE TITLE
Re-Read all data on disk to verify data on disk is intact, not in repository, so vice versa check command

### ISSUE THREAD

**Issue by JsBergbau** [restic/restic#3412]
Output of `restic version`
--------------------------
restic 0.12.0 compiled with go1.15.8 on windows/amd64

What should restic do differently? Which functionality do you think we should add?
----------------------------------------------------------------------------------
There is the "check" command to check that data in the repository is intact.
However also your origin disk can get corrupted so wrong data is read.
Since data inside the repository contains checksums you could use this data to verify that your data on disk is still ok.
Of course there are some changed files because these were really updated. Restic should check only those files that didn't change according to timestamp (as restic does when considering if it should add a file). 

What are you trying to do? What problem would this solve?
---------------------------------------------------------
Data on disks can get corrupted. By using restic backup to check if the files on disk are after years still the same as when they were added to repository you can avoid that your family photos and other data has become corrupted. Normally you would notice that only after years when you open these files again.

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------
Restic is a great tool. It gives me a very good feeling that my data is safe. When my harddrive fails or I accidentally deleted a file restic is there to get my data back.

**Comment by MichaelEischer** [comment]
This looks like a duplicate of either #805 or #1620 to me.

**Comment by JsBergbau** [comment]
Ok thanks. It is a duplicate of https://github.com/restic/restic/issues/805 
#1620 doesn't really help a lot from my point of view. If you verify some restores it should be clear that restic restores the files in the correct order. Verifying restored files means re-reading them, taking a lot of time and if something went wrong there you have a hardware problem which affects also other programs and can be detected with already available software.

**Comment by MichaelEischer** [comment]
Then let's close this issue as duplicate of #805.
